### DISCUSSION TITLE
Gutter changes scrollbar indicator color is confusing

### DISCUSSION THREAD

**Discussion by abdel-ships-it** [gitkraken/vscode-gitlens#1961] [Feedback]
Im not sure when this changed, but since recently the scrollbar indicator has what I think is a new addition. Which is a red indicator for git deletions. Now I don't want to turn off the scrollbar indicator as I find it super useful.

This pertains to the following setting `gitlens.changes.locations` with `overview` value being part of the setting. The color red in the scrollbar area throws me off because usually typescript communicates errors in that area with the same color.

I was wondering if this can be turned off, or if the color can be changed in the settings somehow. That way I can still enjoy being able to quickly scroll to changes, without having to turn the feature completely off. See screenshot below to understand what I'm talking about. Also, I'm on version `12.0.5`. Thanks !

![image](https://user-images.githubusercontent.com/10495548/162761745-be938b87-ad72-44e5-a90f-4a33eab05859.png)

**Comment by eamodio** [comment]
@realappie are you using the GitLens "File Changes" annotation or are you talking about the built-in SCM decorations that VS Code adds for uncommitted changes?

**Comment by abdel-ships-it** [comment]
@eamodio Im talking about uncommitted changes. Not the file changes view 😊
